File: Sample/client/Zezex/Linkd/ModuleLinkd.cs
﻿
using System.Threading.Tasks;
using Zeze.Net;

namespace Zezex.Linkd
{
    public sealed partial class ModuleLinkd : AbstractModule
    {
        public void Start(Client.App app)
        {
        }

        public void Stop(Client.App app)
        {
        }

        protected override async Task<long> ProcessKeepAlive(Protocol _p)
        {
            var p = _p as KeepAlive;
            return Zeze.Transaction.Procedure.NotImplement;
        }
    }
}

File: ZezeJava/ZezexJava/client/src/ClientZezex/Linkd/ModuleLinkd.java
package ClientZezex.Linkd;

public class ModuleLinkd extends AbstractModule {
    public void Start(ClientGame.App app) throws Throwable {
    }

    public void Stop(ClientGame.App app) throws Throwable {
    }

    @Override
    protected long ProcessKeepAlive(ClientZezex.Linkd.KeepAlive p) {
        return Zeze.Transaction.Procedure.NotImplement;
    }

    // ZEZE_FILE_CHUNK {{{ GEN MODULE @formatter:off
    public ModuleLinkd(ClientGame.App app) {
        super(app);
    }
    // ZEZE_FILE_CHUNK }}} GEN MODULE @formatter:on
}
